guide/intro/chapter-1/index.html

```diff
@@ -0,0 +1,88 @@
+---
+layout: default
+title: What is Pygame
+next_page: [/guide/intro/chapter-2, Opening A Window]
+---
+
+<body>
+  <p>In this article, we'll go over what pygame is and how to install it.</p>
+
+  <h2>Table of contents</h2>
+  <p></p>
+  <ol id="toc">
+    <!--
+            To be filled by javascript
+            -->
+  </ol>
+
+  <h2><a id="introduction" class="js">Introduction</a></h2>
+  <p>
+    It's time to get started on your pygame journey, but what is pygame? Well
+    pygame is "a free and open-source cross-platform library for the development
+    of multimedia applications like video games using Python". As the
+    description suggests, it's mostly used to make video games - which is why in
+    this book we'll be learning techniques relevant to making games
+  </p>
+
+  <p>
+    Pygame provides the very basic utility to draw and manipulate pixels on to
+    the screen, much like <a href="https://www.libsdl.org/">SDL2</a>, but with
+    extended features in various forms that makes it easier to use. Given this
+    nature, you generally have a lot of control on how you want your code to
+    run, so it may have a bit of a learning curve. But fear not! After reading
+    this book you should be able to use pygame just fine, as long as you get
+    your practice in 🔫
+  </p>
+
+  <p>
+    In this book we'll aim to cover how to do basic things with pygame, with
+    little exercises in between to make sure you're not falling off, goodluck
+  </p>
+
+  <h2><a id="pygame-ce" class="js">Pygame-CE</a></h2>
```

Comment:
This whole section makes a large number of bold claims, but doesn't provide any supporting evidence for it. I suggest instead linking to some external party dialogues and briefly summarizing the differences instead. The drama part is not really important to the average noob and I suspect the heavy bias here might rub people the wrong way

Reply:
> This whole section makes a large number of bold claims

Which ones?
> I suspect the heavy bias here might rub people the wrong way

Hmm, I think removing the last sentence might fix it: "It is recommended to use only `pygame-ce` from now onwards by most Python communities."
> The drama part is not really important to the average noob

I don't really think I dwelt too deep into the drama, I just mentioned how pygame-ce was formed

Reply:
Maybe this line
"Well, pygame-ce has no "owner", it follows a more democratic approach to managing itself now."

Could be changed to 
"`pygame-ce` is not owned by any one person and follows a more democratic model to its management"

Reply:
So this is how it would be now:
So pygame currently has two major distributors, the original one and a fork of the project. What is a fork? Well it means it's a copy of the project, and in this case they branched off to be two completely separate packages now.

[pygame-ce](https://github.com/pygame-community/pygame-ce) is a fork of the original pygame, which got created due to all the active developers of the original project being banned by the "owner". The fork is not owned by any one person and follows a more democratic model to its management

So what happened to all the active developers of the original pygame? They formed pygame-ce! That's right, so pygame-ce is being developed by the same people (and more) as the original pygame, and with a more democratic model have been able to push releases way more easily and introduce many more features.

For all sane purposes, we will be using pygame-ce in this book and we advise the same. It has had more performance improvements, code features, and active development than the original pygame for a while now.